homepage/de.js

```diff
@@ -0,0 +1,67 @@
+	"Personal pricing": "Pers&ouml;nlicher Preis",
+	"Free": "Kostenlos",
+	"Forever": "F&uuml;r immer",
+	"<b>Unlimited</b> public rooms": "<b>Unbegrenzt</b> &ouml;ffentliche Chatrooms",
+	"<b>1</b> private room": "<b>1</b> privater Chatroom",
+	"<b>2 weeks’</b> chat history": "<b>2 w&ouml;chigee</b> Chat-Verlauf",
+	"<b>Get started</b> for free": "Kostenlos <b>registrieren</b>",
+	"Individual": "einzeln",
+	"monthly": "monatlich",
+	"<b>Unlimited</b> private rooms": "<b>Unbegrenzte</b> private Chatrooms",
+	"<b>Unlimited</b> chat history": "<b>Unbegrenzter</b> Chat-Verlauf",
+	"<b>Purchase</b> now": "Jetzt <b>kaufen</b>",
+	"Organisation pricing": "Preispl&auml;ne",
+	"Bronze": "Bronze",
+	"per user/month": "pro Benutzer/Monat",
+	"<b>6 months’</b> chat history": "<b>6 monatiger</b> Chat-Verlauf",
+	"Silver": "Silber",
+	"Our free plan includes one private room per organisation you belong to on GitHub": "Unser kostenloses Angebot beinhaltet einen kostenlosen Chatroom pro Organisation der du auf Github angeh&ouml;rst",
+	"Our Individual plan allows you to create private rooms using your personal account as the owner": "Unser pers&ouml;nliches Angebot erlaubt dir private Chatrooms zu erstellen, bei denen dein pers&ouml;nlicher Account als Eigent&uuml;mer genutzt wird",
+	"Our organisation plans allow you to create private rooms using your GitHub organisation as the owner": "Unser Angebot f&uuml;r Unternehmen bietet die M&ouml;glichkeit private Chatrooms zu erstellen, bei denen die GitHub-Organisation als Eigent&uuml;mer genutzt wird",
+	"Prices may be subject to VAT. By signing up you agree to our <a %s>Terms and Conditions</a>.": "Alle Preise unterliegen der Mehrwertssteuer. Mit der Registrierung stimmst du unseren <a %s>Nutzungsbedingungen</a> zu.",
+	"Get Gitter now!": "Jetzt Gitter verwenden!",
+	"Gitter is available in all modern browsers as well as apps for desktops and mobile phones.": "Gitter ist in allen modernen Browsern verf&uuml;gbar, wir bieten aber auch Anwendungen f&uuml;r Desktops und Smartphones.",
+	"We currently have apps for OSX, iOS, Android and Windows with a Linux app coming soon. We also have a sweet little <a %s>IRC bridge</a>": "Wir bieten momentan Apps f&uuml;r OSX, iOS, Android, Windows und Linux an. Zudem haben wir auch eine <a %s>IRC-Bridge</a>",
+	"Mac": "Mac",
+	"Windows": "Windows",
+	"Coming soon": "Bald verf&uuml;gbar",
+	"iPhone": "iPhone",
+	"Android": "Android",
+	"By developers, for developers": "Von Entwicklern, f&uuml;r Entwickler",
+	"Gitter is built in London by a team of developers just like you. Hell even the product guy writes code, although we generally end up rewriting those bits.": "Gitter wird in von einem Team von Entwicklern wie dir in London entwickelt. Auch der Produktmanager schreibt Code, obwohl wir meistens diese Bits wieder umschreiben d&uuml;rfen.",
```

Comment:
I’d say „Teile“ instead of „Bits“ in German, since this isn’t about logical bits.